app/validators/image_validator.rb

```diff
@@ -44,8 +44,10 @@ def validate_size(record, image)
     target_width = @size[0]
     target_height = @size[1]
 
-    unless actual_width == target_width && actual_height == target_height
+    if actual_width < target_width || actual_height < target_height
```

Comment:
I changed the number of the function here to make clear that it's checking each dimension individually, let me know if you have any ideas as to how it could be made even clearer